<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FLChooseOutputDialog</class>
 <widget class="QDialog" name="FLChooseOutputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose Output</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="11">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="9" colspan="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>ALL TrbCorrs</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="9">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>502</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="11">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="4" column="3">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>All IptCorrs</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="11">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Outputs</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>All Local TRB</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QCheckBox" name="checkBox_4">
     <property name="text">
      <string>All StatCorrs</string>
     </property>
    </widget>
   </item>
   <item row="4" column="11">
    <widget class="QCheckBox" name="checkBox_5">
     <property name="text">
      <string>All RemIptCorrs</string>
     </property>
    </widget>
   </item>
   <item row="4" column="10">
    <widget class="QCheckBox" name="checkBox_6">
     <property name="text">
      <string>All Relay</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
